source/apiVolontaria/apiVolontaria/settings.py

```diff
@@ -126,6 +128,22 @@
 
 USE_TZ = True
 
+LANGUAGES = [
+    ('en', _('English')),
+    ('fr', _('French')),
+]
```

Comment:
I don't think we should restrict the API to only english and french. If custom messages are not translated in the language of choice of the user, then english will be used by default, but all default messages will be received in the preferred language of the user.

Reply:
I think you're right. We prefer a partial translation than a complete limitation.